packages/orbit-components/src/Stepper/StepperStateless/index.tsx

```diff
@@ -1,118 +1,34 @@
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import mq from "../../utils/mediaQuery";
 import Minus from "../../icons/Minus";
 import Plus from "../../icons/Plus";
-import defaultTheme from "../../defaultTheme";
-import Button from "../../primitives/ButtonPrimitive";
+import ButtonPrimitive from "../../primitives/ButtonPrimitive";
 import useTheme from "../../hooks/useTheme";
-import { defaultFocus } from "../../utils/common";
 import type { Props } from "./types";
 
-const getMaxWidth = ({ maxWidth }) => {
+const getMaxWidth = ({ maxWidth }: { maxWidth: string | number }) => {
   if (typeof maxWidth === "string") return maxWidth;
-  return `${parseInt(maxWidth, 10)}px`;
+  return `${parseInt(maxWidth.toString(), 10)}px`;
 };
 
-const StyledStepper = styled.div`
-  display: flex;
-  width: 100%;
-  max-width: ${getMaxWidth};
-  flex: 1 1 auto;
-`;
-
-const iconMixin = css<{ isActive?: boolean; isDisabled?: boolean }>`
-  ${({ theme, isActive, isDisabled }) => css`
-    padding: 2px;
-    height: 20px;
-    width: 20px;
-    background: ${isActive ? theme.orbit.paletteBlueNormal : theme.orbit.paletteCloudNormal};
-    border-radius: ${theme.orbit.borderRadiusCircle};
-    ${mq.desktop(css`
-      height: 16px;
-      width: 16px;
-    `)};
-
-    ${!isDisabled &&
-    css`
-      &:hover {
-        background: ${isActive
-          ? theme.orbit.paletteBlueNormalHover
-          : theme.orbit.paletteCloudNormalHover};
-      }
-
-      &:focus,
-      &:active {
-        box-shadow: inset 0 0 0 2px
-          ${isActive ? theme.orbit.paletteBlueLightActive : theme.orbit.paletteCloudNormalActive};
-      }
-    `};
-  `}
-`;
-
-const StyledMinusIcon = styled(Minus)`
-  ${iconMixin};
-`;
-
-StyledMinusIcon.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledPlusIcon = styled(Plus)`
-  ${iconMixin};
-`;
-
-StyledPlusIcon.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledStepperInput = styled.input`
-  ${({ theme, disabled }) => css`
-    width: 100%;
-    height: 44px;
-    padding: 0;
-    border: 0;
-    font-size: ${theme.orbit.fontSizeTextLarge};
-    font-weight: ${theme.orbit.fontWeightMedium};
-    color: ${theme.orbit.paletteInkDark};
-    text-align: center;
-    min-width: 0;
-    opacity: ${disabled ? "0.5" : "1"};
-    background-color: ${disabled && "transparent"};
-
-    &::-webkit-inner-spin-button,
-    &::-webkit-outer-spin-button {
-      -webkit-appearance: none;
-      margin: 0;
-    }
-
-    &:focus {
-      outline: none;
-    }
-  `}
-`;
-
-StyledStepperInput.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledStepperButton = styled(Button)`
-  &:focus {
-    outline: none;
-
-    ${StyledMinusIcon},
-    ${StyledPlusIcon} {
-      ${defaultFocus}
-    }
-  }
-`;
-
-StyledStepperButton.defaultProps = {
-  theme: defaultTheme,
-};
+const stepperButtonMixin = ({ disabled, active }: { disabled: boolean; active?: boolean }) =>
+  cx(
+    "bg-transparent",
```

Comment:
Minor: Is this necessary? 

It doesn't harm, but I believe Tailwind has transparent bg in buttons by default.

Reply:
hm, not tailwind, but our primitive, so ok, can be removed, thanks!